main.js

```diff
@@ -104,47 +55,52 @@ const editListItem = (event, id) => {
     editionInput.addEventListener('change', editionEvent => {
         li.textContent = editionEvent.target.value
         todos = todos.map(todo => {
-            if (id === todo.id) {
-                todo.text = li.textContent
+            if (id === todo.getId()) {
+                todo.setText(li.textContent)
+
                 return todo
             }
+
             return todo
         })
-        div.classList.remove('edited-list-item')
+        parentDiv.classList.remove('edited-list-item')
         li.classList.remove('edited-list-item__li-tag')
-        div.classList.add('list-item')
+        parentDiv.classList.add('list-item')
         li.classList.add('list-item__li-tag')
         span.style.display = 'block'
     })
 }
 
-
 const filterList = event => {
     const currentNodeList = document.querySelectorAll('.list-item')
-    const finishedTasksList = todos.filter(todo => todo.isDone)
-    const finishedId = finishedTasksList.map(task => task.id)
-    console.log(finishedId)
-    console.log(finishedTasksList)
-    console.log(currentNodeList)
+    const finishedTasksIds = todos
+        .filter(todo => todo.getIsDone())
+        .map(task => task.getId())
 
     if (event.target.id === 'todo') {
         currentNodeList.forEach(div => {
-            if (finishedId.includes(div.getAttribute('todo-id'))) {
+            if (finishedTasksIds
+                .includes(div.getAttribute('todo-id'))) {
                 div.style.display = 'none'
-            } else {
-                div.style.display = 'flex'
+
+                return
             }
+                div.style.display = 'flex'
         })
     }
+
     if (event.target.id === 'strokes') {
         currentNodeList.forEach(div => {
-            if (!finishedId.includes(div.getAttribute('todo-id'))) {
+            if (!finishedTasksIds
+                .includes(div.getAttribute('todo-id'))) {
```

Comment:
I don't like it. You have a class with `#id` inside. You should also have reference to the todo element like `#ref`.
You can find class instances with `isDone` flag:
```
const finishedTasks = todos
         .filter(todo => todo.getIsDone())
         .map(todo => todo.getRef())
```
and get reference to every item

Reply:
but what should be that #ref?